### PR TITLE
JESI-3634

### PR DESCRIPTION
**Pull request by AndreTheHunter**
[JESI-3634](https://jesims.atlassian.net/browse/JESI-3634)
- [ ] https://github.com/jesims/bindle/pull/31

### CODE REVIEW

File: package-dry.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "name": "customs",
```
Comment (AndreTheHunter) [inline package-dry.json:2]: Random fix. NPM now does not like having the same name

File: .circleci/config.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ aliases:
         - /home/node/.lein
         - /home/node/.m2
     node_cache: &NODE_CACHE
-      key: node-{{ .Environment.CACHE_VERSION }}-{{ checksum "/tmp/.node_version" }}-{{ checksum "package-lock.json" }}
+      key: node-{{ .Environment.CACHE_VERSION }}-{{ .Environment.NODE_VERSION }}-{{ checksum "package-lock.json" }}
```
Comment (AndreTheHunter) [inline .circleci/config.yml:11]: Random fix. We don't create `.node_version` any more

File: src/io/jesi/customs/kaocha.clj [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  (:require
+    [io.jesi.customs.util :as util])
+  (:import
+    (java.io ByteArrayOutputStream)))
+
+(defn reset-output []
+  (when (and (util/provided? 'kaocha.plugin.capture-output)
+             ;deref var and atom
+             (seq @@(resolve 'kaocha.plugin.capture-output/active-buffers)))
+    (when-let [^ByteArrayOutputStream test-buffer @(resolve 'kaocha.plugin.capture-output/*test-buffer*)]
```
Comment (AndreTheHunter) [inline src/io/jesi/customs/kaocha.clj:11]: Some lazy loading shenanigans

File: src/io/jesi/customs/kaocha.clj [hunk 4]
```diff
@@ -0,0 +1,12 @@
+(ns io.jesi.customs.kaocha
+  (:require
+    [io.jesi.customs.util :as util])
+  (:import
+    (java.io ByteArrayOutputStream)))
+
+(defn reset-output []
```
Comment (AndreTheHunter) [inline src/io/jesi/customs/kaocha.clj:7]: I got sick of doing `(.reset kaocha.plugin.capture-output/*test-buffer*)`

**Review by axrs — APPROVED** [review]

**Pull request merged**